test/app/digest.test.lua

```diff
@@ -203,3 +201,66 @@ _ = fiber.create(test_pbkdf2)
 _ = sentry:get()
 _ = sentry:get()
 res
+
+--
+-- gh-2003 xxHash
+--
+xxhash32 = digest.xxhash32.new()
+xxhash32:result()
+xxhash64 = digest.xxhash64.new()
+xxhash64:result()
+
+-- new takes seed optionally
```

Comment:
From https://github.com/tarantool/tarantool/wiki/Code-review-procedure : "Start sentences from a capital letter, end with a **dot**." Here and below.

Reply:
Fixed. Thanks!